src/templates/about-page.js

```diff
@@ -82,4 +82,4 @@ export const aboutPageQuery = graphql`
       }
     }
   }
-`
+`
```

Comment:
Kinda weird there's no newline here. It's probably a windows thing - I'll still merge it.